lib/mwoffliner.lib.js

```diff
@@ -25,10 +25,11 @@ var os = require( 'os' );
 var crypto = require( 'crypto' );
 var unicodeCutter = require( 'utf8-binary-cutter' );
 var htmlMinifier = require('html-minifier');
+const fetch = require('node-fetch')
 
 module.exports = {
     getParametersList: () => [
-        { name: 'mwUrl', description: 'Mediawiki base URL', required: true },
+        { name: 'mwUrl', description: 'Mediawiki base URL. Dont forget the trailing /', required: true },
```

Comment:
This point is not related to the current ticket. @Skylsmoi improved the text because he faced the problem. We can fix it during next developements, please open another issue.